modules/installation/pages/install-ibmz.adoc

```diff
@@ -90,6 +90,8 @@ The hostname should contain less than eight characters.
 
 This section covers the installation of {productname}{nbsp}{productnumber} as an extension to {sls}{nbsp}{sles-version}{nbsp}{sp-version}.
 
+TBD: Is this correct? SUSE Manager as an extension? Doesn't SLES15SP1 on z also have the new module concept with SUSE Manager being a base product?
```

Comment:
```suggestion
// TBD: Is this correct? SUSE Manager as an extension? Doesn't SLES15SP1 on z also have the new module concept with SUSE Manager being a base product?
```

Putting this in a comment to avoid accidents :wink: